content/docs/0.6.0/reference/cli/index.md

```diff
@@ -111,9 +109,9 @@ keptn configure domain YOUR.CUSTOM.DOMAIN
 
 ### keptn auth 
 
-Before the keptn CLI can be used, it needs to be authenticated against a keptn cluster. Therefore, an endpoint and an API token are required.
+Before the Keptn CLI can be used, it needs to be authenticated against a Keptn cluster. Therefore, an endpoint and an API token are required.
```

Comment:
I thought if we are talking about Keptn as the CLI, we use a lowercase k?

Reply:
That is true. When talking about the keptn CLI, we use lower case k. I fixed it.